app/main.py

```diff
@@ -20,10 +27,114 @@
 app.include_router(email.router)
 
 
+def add_countries_to_db():
+    """
+    Adding all new countries to the "Country" table in the database.
+    Information is based on the "countries" list.
+    (The list is located in app/database/countries.py)
+    Names are described either as:
+    "Country Name, City Name" or
+    "Country Name" solely.
+    Timezones are described as "Continent/ City Name"
+    for example:
+        name: Israel, Jerusalem
+        timezone: Asia/Jerusalem
+    """
+    session = SessionLocal()
+    for country in countries:
+        partial_name = country['name']
+        for capital in country['timezones']:
+            capital_name = capital.split('/')[-1]
+            if partial_name != capital_name:
+                name = partial_name + ', ' + capital_name
+            else:
+                name = capital_name
+            existing = session.query(Country).filter_by(name=name).first()
+            if not existing:
+                new_country = Country(name=name, timezone=str(capital))
+                session.add(new_country)
+    session.commit()
+    session.close()
+
+
+add_countries_to_db()
+
+
 @app.get("/")
-async def home(request: Request):
+async def home(
+        request: Request):
     return templates.TemplateResponse("home.html", {
         "request": request,
         "message": "Hello, World!"
+    })
+
 
+@app.get("/global_time")
+async def time(
+        request: Request):
+    """
+    Displays a globe icon.
+    By Click the page transfers to "/global_time/choose".
+    """
+    return templates.TemplateResponse("event/partials/global_time.html", {
+        "request": request
     })
+
+
+@app.get("/global_time/choose")
+async def time_choose(
+        request: Request,
+        session=Depends(get_db)):
+    """
+    Displays the list of all countries name from "Country" table.
+    By Click on country name the page transfers to:
+    "/global_time/{country_name}/{chosen_datetime}".
+    *** Temporarily  returns a fictitious meeting datetime. ***
+    """
+    data = session.query(Country.name).all()
```

Comment:
I think this one should be on a separate function, cached by `@functools.lru_cache`.

The countries probably won't change, and querying the database every time can to be a bit expensive :)